src/napari_imagej/config_default.yaml

```diff
@@ -19,7 +19,7 @@ imagej_base_directory: '.'
 # This can be used to include original ImageJ functionality.
 # Iff true, original ImageJ functionality (ij.* packages) will be available.
 # Defaults to false.
-include_imagej_legacy: false
+include_imagej_legacy: true
```

Comment:
What is the reasoning for this change?

Assuming we do this, we also need to change the documentation.

Reply:
Wasn't the point of the hack `napari_imagej/java` to get around that issue? Or am I missing something?

Reply:
We made `add_legacy=False` before we added the GUI buttons, right? Now that we have the GUI buttons, bringing up the unfinished Swing UI by default strikes me as rather rude; people are very used to the legacy UI, and it is much better tested, and the FlatLaf bug doesn't happen with that UI.

Reply:
> I agree that there are challenges to solve if we change this, particularly [imagej/imagej-legacy#280](https://github.com/imagej/imagej-legacy/issues/280).

I thought this was the main reason that we chose to use pure IJ2. See #58

The other reason that we chose to use pure IJ2 is that closing the legacy UI terminates napari, as described in #93.

Tbh, I don't really like either option here.

Reply:
Also, if we do decide to make legacy the default, we should rewrite the conda job testing legacy to test pure IJ2.

Reply:
Actually, if you close the IJ2 UI on this branch, you terminate napari. So that stinks, and we have to get to the bottom of that...

Reply:
> Actually, if you close the IJ2 UI on this branch, you terminate napari. So that stinks, and we have to get to the bottom of that...
Okay, this is fixed with 008daec.